src/account/reducer.ts

```
@@ -18,15 +18,15 @@ export interface State {
   pincodeType: PincodeType
   backupCompleted: boolean
   accountCreationTime: number
-  startOnboardingTime: number | undefined
+  startOnboardingTime?: number
   dismissedGetVerified: boolean
   dismissedGoldEducation: boolean
   acceptedTerms: boolean
   hasMigratedToNewBip39: boolean
-  choseToRestoreAccount: boolean | undefined
-  profileUploaded: boolean | undefined
-  recoveringFromStoreWipe: boolean | undefined
-  accountToRecoverFromStoreWipe: string | undefined
+  choseToRestoreAccount?: boolean
```

new versions of ts-json-schema-generator do not recognize the `| undefined`

---

The rootstate should fail as long as the base state for the field is undefined